symplyphysics/laws/relativistic/vector/relativistic_velocity_parallel_to_movement.py

```diff
@@ -0,0 +1,70 @@
+from sympy.physics.units import speed_of_light
+from symplyphysics import (
+    units,
+    Quantity,
+    validate_input,
+    validate_output,
+    Vector,
+    dot_vectors,
+    add_cartesian_vectors,
+    scale_vector,
+    QuantityVector,
+    cross_cartesian_vectors,
+    vector_magnitude,
+)
+
+# Description
+## Consider two inertial reference frames: one fixed (lab frame) and one tied to the moving object (proper frame).
+## The proper frame is moving with some velocity `v` relative to lab frame. Then, according to the theory of special
+## relativity, the velocity of the object relative to lab frame is not equal to the sum of its velocity in the proper
+## frame and the velocity of the proper frame relative to the lab frame.
+
+# Law: u_parallel = (u_parallel' + v) / (1 + dot(u_parallel', v) / c**2)
+## u_parallel - velocity vector relative to lab frame in the perpendicular direction
```

Comment:
Cannot understand, what is perpendicular here. And why it is called `parallel` if it's in perpendicular direction.